src/app/core/auth.service.ts

```diff
@@ -86,24 +86,21 @@ export class AuthService {
    * this method assumes the user has just been successfully
    * linked back to the homepage through SSO and depends on cookies
    * set by the SSO routehandler
+   *
+   * @param cookieString the cookie set by the SSO routehandler
+   * @memberof AuthService
    */
-  public setSsoSession() {
-    const identityId = this.cookies.get('identityId');
-    const token = this.cookies.get('token');
-    const domain = environment.production ? 'clark.center' : 'localhost';
-    this.cookies.delete('identityId', '/', domain, false, 'Lax');
-    this.cookies.delete('token', '/', domain, false, 'Lax');
-    const sessionObject = {
-      user: this.user,
-      tokens: {
-        bearer: this.cookies.get('presence'),
-        openId:{
-          IdentityId: identityId,
-          Token: token
-        }
-      }
-    };
-    this.setSession(sessionObject);
+  public setSsoSession(cookieString: string) {
+    const token = JSON.parse(cookieString);
+      const user = token.user;
+      const tokens: Tokens = {bearer: token.bearer, openId: token.openId};
```

Comment:
is openId being used anymore?

Reply:
Paige says it's for encryption that isn't really needed anymore but it's a required param for setSession() and will break something if we remove it